Build is failing on Windows

I've tried to build the latest version of the project on Windows 10 running this command:
```
mvn clean install -Pqulice
```
but I got 6 test failures. 5 failed tests are:

- com.zerocracy.tk.project.TkArtifactTest#rendersHomePage
- com.zerocracy.tk.project.TkFootprintTest#rendersListOfClaims
- com.zerocracy.pm.cost.EquityTest#addValue
- com.zerocracy.tk.TkAppTest#rendersHomePageViaHttp
- com.zerocracy.tk.profile.TkIdentifyTest#rendersHomePage

and have the same assertion error:
```
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.871 sec <<< FAILURE! - in com.zerocracy.tk.profile.TkIdentifyTest
rendersHomePage(com.zerocracy.tk.profile.TkIdentifyTest)  Time elapsed: 0.871 sec  <<< FAILURE!
java.lang.AssertionError: 

Expected: (an XML document with XPath //xhtml:article)
     but: an XML document with XPath //xhtml:article was <<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <meta charset="utf-8"/>
        <title>0crat: error</title>
        <meta name="description" content="Internal application error"/>
        <meta name="viewport" content="width=device-width,initial-scale=1.0"/>
        <link rel="shortcut icon" href="/png/logo.png"/>
        <link rel="stylesheet" href="/css/main.css"/>
        <title>oops...</title>
    </head>
    <body>
        <p>
            <a href="/">
                <img src="/svg/logo.svg" class="logo" />
            </a>
        </p>
        <h1>Internal application error</h1>
        <p>1.0-SNAPSHOT, </p>
        <p>
            Please, report it
            <a href="https://github.com/zerocracy/farm/issues">here</a>.
            We will do our best to fix it as soon as possible.
            We're very sorry.
        </p>
        <pre>java.lang.IllegalStateException: [GET http://www.example.com/identify] failed in 35ms: ValidationException: Cannot convert string &quot;-1,000&quot; to double
	at org.takes.facets.fallback.TkFallback.error(TkFallback.java:234)
	at org.takes.facets.fallback.TkFallback.access$100(TkFallback.java:54)
	at org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:202)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsPrint.printBody(RsPrint.java:161)
	at org.takes.rs.RsPrint.printBody(RsPrint.java:91)
	at com.zerocracy.tk.profile.TkIdentifyTest.rendersHomePage(TkIdentifyTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: ValidationException: Cannot convert string &quot;-1,000&quot; to double
	at org.takes.rs.RsXslt.transform(RsXslt.java:133)
	at org.takes.rs.RsXslt.access$100(RsXslt.java:89)
	at org.takes.rs.RsXslt$1.body(RsXslt.java:113)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.fork.RsFork$1.body(RsFork.java:74)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at com.zerocracy.tk.TkSoftForward$RsSoft$1.body(TkSoftForward.java:95)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.forward.TkForward$Safe.load(TkForward.java:116)
	at org.takes.facets.forward.TkForward$Safe.body(TkForward.java:102)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.fork.RsFork$1.body(RsFork.java:74)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeaders$1.body(RsWithHeaders.java:78)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:187)
	... 22 more
Caused by: ValidationException: Cannot convert string &quot;-1,000&quot; to double
	at net.sf.saxon.type.ValidationFailure.makeException(ValidationFailure.java:393)
	at net.sf.saxon.type.ValidationFailure.asAtomic(ValidationFailure.java:419)
	at net.sf.saxon.expr.sort.UntypedNumericComparer.quickComparison(UntypedNumericComparer.java:159)
	at net.sf.saxon.expr.sort.UntypedNumericComparer.quickCompare(UntypedNumericComparer.java:59)
	at net.sf.saxon.expr.GeneralComparison.compare(GeneralComparison.java:833)
	at net.sf.saxon.expr.GeneralComparison.evaluateManyToOne(GeneralComparison.java:711)
	at net.sf.saxon.expr.GeneralComparison.effectiveBooleanValue(GeneralComparison.java:660)
	at net.sf.saxon.expr.instruct.Choose.choose(Choose.java:902)
	at net.sf.saxon.expr.instruct.Choose.iterate(Choose.java:953)
	at net.sf.saxon.expr.instruct.BlockIterator.next(BlockIterator.java:49)
	at net.sf.saxon.tree.iter.AdjacentTextNodeMergingIterator.next(AdjacentTextNodeMergingIterator.java:52)
	at net.sf.saxon.tree.iter.UntypedAtomizingIterator.next(UntypedAtomizingIterator.java:50)
	at net.sf.saxon.tree.iter.UntypedAtomizingIterator.next(UntypedAtomizingIterator.java:32)
	at net.sf.saxon.expr.ItemMappingIterator.next(ItemMappingIterator.java:108)
	at net.sf.saxon.functions.FoldingFunction.call(FoldingFunction.java:59)
	at net.sf.saxon.expr.FunctionCall.iterate(FunctionCall.java:547)
	at net.sf.saxon.expr.Expression.evaluateItem(Expression.java:831)
	at net.sf.saxon.expr.Expression.evaluateAsString(Expression.java:890)
	at net.sf.saxon.expr.instruct.SimpleNodeConstructor.processLeavingTail(SimpleNodeConstructor.java:218)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.NamedTemplate.expand(NamedTemplate.java:262)
	at net.sf.saxon.expr.instruct.CallTemplate.process(CallTemplate.java:353)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)
	at net.sf.saxon.expr.instruct.ApplyTemplates.apply(ApplyTemplates.java:295)
	at net.sf.saxon.expr.instruct.ApplyTemplates.processLeavingTail(ApplyTemplates.java:252)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)
	at net.sf.saxon.trans.TextOnlyCopyRuleSet.process(TextOnlyCopyRuleSet.java:66)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:478)
	at net.sf.saxon.Controller.transformDocument(Controller.java:2402)
	at net.sf.saxon.Controller.transform(Controller.java:1970)
	at net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:596)
	at net.sf.saxon.jaxp.TransformerImpl.transform(TransformerImpl.java:73)
	at org.takes.rs.RsXslt.transform(RsXslt.java:160)
	at org.takes.rs.RsXslt.transform(RsXslt.java:131)
	... 44 more
</pre>
    </body>
</html>
>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at com.zerocracy.tk.profile.TkIdentifyTest.rendersHomePage(TkIdentifyTest.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

And then there is the 6th failed test:

- com.zerocracy.farm.S3ItemTest#refreshesFilesOnServer

that has another cause:
```
Tests run: 5, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 0.122 sec <<< FAILURE! - in com.zerocracy.farm.S3ItemTest
refreshesFilesOnServer(com.zerocracy.farm.S3ItemTest)  Time elapsed: 0.121 sec  <<< FAILURE!
java.lang.AssertionError: 

Expected: not an empty iterable
     but: was <[]>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at com.zerocracy.farm.S3ItemTest.refreshesFilesOnServer(S3ItemTest.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

I guess it's somehow related to my current locale that is the following:
![image](https://user-images.githubusercontent.com/3031372/40388406-1c269e46-5e18-11e8-91e8-26b0875b1a30.png)

**TODO:**
- fix the build on Windows OS

@g4s8 I've tried to build the latest master with command `mvn clean install --batch-mode -Pqulice` and got a few failures:
```
Failed tests: 
  TkBoardTest.rendersBoardWithFundedProject:69 
Expected: (an XML document with XPath //xhtml:span[@title = 'The project is funded' and . = '$256.00'])
     but: an XML document with XPath //xhtml:span[@title = 'The project is funded' and . = '$256.00'] was "<!DOCTYPE html>\n<html xmlns=\"http://www.w3.org/1999/xhtml\">\n    <head>\n        <meta charset=\"utf-8\"/>\n        <title>0crat: error</title>\n        <meta name=\"description\" content=\"Internal application error\"/>\n        <meta name=\"viewport\" content=\"width=device-width,initial-scale=1.0\"/>\n        <link rel=\"shortcut icon\" href=\"/png/logo.png\"/>\n        <link rel=\"stylesheet\" href=\"/css/main.css\"/>\n        <title>oops...</title>\n    </head>\n    <body>\n        <p>\n            <a href=\"/\">\n                <img src=\"/svg/logo.svg\" class=\"logo\" />\n            </a>\n        </p>\n        <h1>Internal application error</h1>\n        <p>1.0-SNAPSHOT, </p>\n        <p>\n            Please, report it\n            <a href=\"https://github.com/zerocracy/farm/issues\">here</a>.\n            We will do our best to fix it as soon as possible.\n            We're very sorry.\n        </p>\n        <pre>java.lang.IllegalStateException: [GET http://www.example.com/board] failed in 194ms: ValidationException: Cannot convert string &quot;-1,000&quot; to double\r\n\tat org.takes.facets.fallback.TkFallback.error(TkFallback.java:234)\r\n\tat org.takes.facets.fallback.TkFallback.access$100(TkFallback.java:54)\r\n\tat org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:202)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsPrint.printBody(RsPrint.java:161)\r\n\tat org.takes.rs.RsPrint.printBody(RsPrint.java:91)\r\n\tat com.zerocracy.tk.TkBoardTest.firefoxView(TkBoardTest.java:120)\r\n\tat com.zerocracy.tk.TkBoardTest.rendersBoardWithFundedProject(TkBoardTest.java:70)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\r\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\r\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\r\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\r\n\tat org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)\r\n\tat org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)\r\n\tat org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:266)\r\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\r\n\tat java.lang.Thread.run(Thread.java:748)\r\nCaused by: java.io.IOException: ValidationException: Cannot convert string &quot;-1,000&quot; to double\r\n\tat org.takes.rs.RsXslt.transform(RsXslt.java:133)\r\n\tat org.takes.rs.RsXslt.access$100(RsXslt.java:89)\r\n\tat org.takes.rs.RsXslt$1.body(RsXslt.java:113)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.facets.fork.RsFork$1.body(RsFork.java:74)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat com.zerocracy.tk.TkSoftForward$RsSoft$1.body(TkSoftForward.java:95)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.facets.forward.TkForward$Safe.load(TkForward.java:116)\r\n\tat org.takes.facets.forward.TkForward$Safe.body(TkForward.java:102)\r\n\tat org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.facets.fork.RsFork$1.body(RsFork.java:74)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWithHeaders$1.body(RsWithHeaders.java:78)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:187)\r\n\t... 23 more\r\nCaused by: ValidationException: Cannot convert string &quot;-1,000&quot; to double\r\n\tat net.sf.saxon.type.ValidationFailure.makeException(ValidationFailure.java:393)\r\n\tat net.sf.saxon.type.ValidationFailure.asAtomic(ValidationFailure.java:419)\r\n\tat net.sf.saxon.expr.sort.UntypedNumericComparer.quickComparison(UntypedNumericComparer.java:159)\r\n\tat net.sf.saxon.expr.sort.UntypedNumericComparer.quickCompare(UntypedNumericComparer.java:59)\r\n\tat net.sf.saxon.expr.GeneralComparison.compare(GeneralComparison.java:833)\r\n\tat net.sf.saxon.expr.GeneralComparison.evaluateManyToOne(GeneralComparison.java:711)\r\n\tat net.sf.saxon.expr.GeneralComparison.effectiveBooleanValue(GeneralComparison.java:660)\r\n\tat net.sf.saxon.expr.instruct.Choose.choose(Choose.java:902)\r\n\tat net.sf.saxon.expr.instruct.Choose.iterate(Choose.java:953)\r\n\tat net.sf.saxon.expr.instruct.BlockIterator.next(BlockIterator.java:49)\r\n\tat net.sf.saxon.tree.iter.AdjacentTextNodeMergingIterator.next(AdjacentTextNodeMergingIterator.java:52)\r\n\tat net.sf.saxon.tree.iter.UntypedAtomizingIterator.next(UntypedAtomizingIterator.java:50)\r\n\tat net.sf.saxon.tree.iter.UntypedAtomizingIterator.next(UntypedAtomizingIterator.java:32)\r\n\tat net.sf.saxon.expr.ItemMappingIterator.next(ItemMappingIterator.java:108)\r\n\tat net.sf.saxon.functions.FoldingFunction.call(FoldingFunction.java:59)\r\n\tat net.sf.saxon.expr.FunctionCall.iterate(FunctionCall.java:547)\r\n\tat net.sf.saxon.expr.Expression.evaluateItem(Expression.java:831)\r\n\tat net.sf.saxon.expr.Expression.evaluateAsString(Expression.java:890)\r\n\tat net.sf.saxon.expr.instruct.SimpleNodeConstructor.processLeavingTail(SimpleNodeConstructor.java:218)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.NamedTemplate.expand(NamedTemplate.java:262)\r\n\tat net.sf.saxon.expr.instruct.CallTemplate.process(CallTemplate.java:353)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)\r\n\tat net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)\r\n\tat net.sf.saxon.expr.instruct.ApplyTemplates.apply(ApplyTemplates.java:295)\r\n\tat net.sf.saxon.expr.instruct.ApplyTemplates.processLeavingTail(ApplyTemplates.java:252)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)\r\n\tat net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)\r\n\tat net.sf.saxon.trans.TextOnlyCopyRuleSet.process(TextOnlyCopyRuleSet.java:66)\r\n\tat net.sf.saxon.trans.Mode.applyTemplates(Mode.java:478)\r\n\tat net.sf.saxon.Controller.transformDocument(Controller.java:2402)\r\n\tat net.sf.saxon.Controller.transform(Controller.java:1970)\r\n\tat net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:596)\r\n\tat net.sf.saxon.jaxp.TransformerImpl.transform(TransformerImpl.java:73)\r\n\tat org.takes.rs.RsXslt.transform(RsXslt.java:160)\r\n\tat org.takes.rs.RsXslt.transform(RsXslt.java:131)\r\n\t... 45 more\r\n</pre>\n    </body>\n</html>\n"
  TkBoardTest.rendersBoardWithProjectWithoutFunding:94 
Expected: (an XML document with XPath //xhtml:span[@title = 'The project has no funds, you will work for free'])
     but: an XML document with XPath //xhtml:span[@title = 'The project has no funds, you will work for free'] was "<!DOCTYPE html>\n<html xmlns=\"http://www.w3.org/1999/xhtml\">\n    <head>\n        <meta charset=\"utf-8\"/>\n        <title>0crat: error</title>\n        <meta name=\"description\" content=\"Internal application error\"/>\n        <meta name=\"viewport\" content=\"width=device-width,initial-scale=1.0\"/>\n        <link rel=\"shortcut icon\" href=\"/png/logo.png\"/>\n        <link rel=\"stylesheet\" href=\"/css/main.css\"/>\n        <title>oops...</title>\n    </head>\n    <body>\n        <p>\n            <a href=\"/\">\n                <img src=\"/svg/logo.svg\" class=\"logo\" />\n            </a>\n        </p>\n        <h1>Internal application error</h1>\n        <p>1.0-SNAPSHOT, </p>\n        <p>\n            Please, report it\n            <a href=\"https://github.com/zerocracy/farm/issues\">here</a>.\n            We will do our best to fix it as soon as possible.\n            We're very sorry.\n        </p>\n        <pre>java.lang.IllegalStateException: [GET http://www.example.com/board] failed in 226ms: ValidationException: Cannot convert string &quot;-1,000&quot; to double\r\n\tat org.takes.facets.fallback.TkFallback.error(TkFallback.java:234)\r\n\tat org.takes.facets.fallback.TkFallback.access$100(TkFallback.java:54)\r\n\tat org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:202)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsPrint.printBody(RsPrint.java:161)\r\n\tat org.takes.rs.RsPrint.printBody(RsPrint.java:91)\r\n\tat com.zerocracy.tk.TkBoardTest.firefoxView(TkBoardTest.java:120)\r\n\tat com.zerocracy.tk.TkBoardTest.rendersBoardWithProjectWithoutFunding(TkBoardTest.java:95)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)\r\n\tat org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)\r\n\tat org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)\r\n\tat org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)\r\n\tat org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)\r\n\tat org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)\r\n\tat org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)\r\n\tat org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)\r\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:266)\r\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\r\n\tat java.lang.Thread.run(Thread.java:748)\r\nCaused by: java.io.IOException: ValidationException: Cannot convert string &quot;-1,000&quot; to double\r\n\tat org.takes.rs.RsXslt.transform(RsXslt.java:133)\r\n\tat org.takes.rs.RsXslt.access$100(RsXslt.java:89)\r\n\tat org.takes.rs.RsXslt$1.body(RsXslt.java:113)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.facets.fork.RsFork$1.body(RsFork.java:74)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat com.zerocracy.tk.TkSoftForward$RsSoft$1.body(TkSoftForward.java:95)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.facets.forward.TkForward$Safe.load(TkForward.java:116)\r\n\tat org.takes.facets.forward.TkForward$Safe.body(TkForward.java:102)\r\n\tat org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.facets.fork.RsFork$1.body(RsFork.java:74)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.rs.RsWithHeaders$1.body(RsWithHeaders.java:78)\r\n\tat org.takes.rs.RsWrap.body(RsWrap.java:65)\r\n\tat org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:187)\r\n\t... 23 more\r\nCaused by: ValidationException: Cannot convert string &quot;-1,000&quot; to double\r\n\tat net.sf.saxon.type.ValidationFailure.makeException(ValidationFailure.java:393)\r\n\tat net.sf.saxon.type.ValidationFailure.asAtomic(ValidationFailure.java:419)\r\n\tat net.sf.saxon.expr.sort.UntypedNumericComparer.quickComparison(UntypedNumericComparer.java:159)\r\n\tat net.sf.saxon.expr.sort.UntypedNumericComparer.quickCompare(UntypedNumericComparer.java:59)\r\n\tat net.sf.saxon.expr.GeneralComparison.compare(GeneralComparison.java:833)\r\n\tat net.sf.saxon.expr.GeneralComparison.evaluateManyToOne(GeneralComparison.java:711)\r\n\tat net.sf.saxon.expr.GeneralComparison.effectiveBooleanValue(GeneralComparison.java:660)\r\n\tat net.sf.saxon.expr.instruct.Choose.choose(Choose.java:902)\r\n\tat net.sf.saxon.expr.instruct.Choose.iterate(Choose.java:953)\r\n\tat net.sf.saxon.expr.instruct.BlockIterator.next(BlockIterator.java:49)\r\n\tat net.sf.saxon.tree.iter.AdjacentTextNodeMergingIterator.next(AdjacentTextNodeMergingIterator.java:52)\r\n\tat net.sf.saxon.tree.iter.UntypedAtomizingIterator.next(UntypedAtomizingIterator.java:50)\r\n\tat net.sf.saxon.tree.iter.UntypedAtomizingIterator.next(UntypedAtomizingIterator.java:32)\r\n\tat net.sf.saxon.expr.ItemMappingIterator.next(ItemMappingIterator.java:108)\r\n\tat net.sf.saxon.functions.FoldingFunction.call(FoldingFunction.java:59)\r\n\tat net.sf.saxon.expr.FunctionCall.iterate(FunctionCall.java:547)\r\n\tat net.sf.saxon.expr.Expression.evaluateItem(Expression.java:831)\r\n\tat net.sf.saxon.expr.Expression.evaluateAsString(Expression.java:890)\r\n\tat net.sf.saxon.expr.instruct.SimpleNodeConstructor.processLeavingTail(SimpleNodeConstructor.java:218)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.NamedTemplate.expand(NamedTemplate.java:262)\r\n\tat net.sf.saxon.expr.instruct.CallTemplate.process(CallTemplate.java:353)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)\r\n\tat net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)\r\n\tat net.sf.saxon.expr.instruct.ApplyTemplates.apply(ApplyTemplates.java:295)\r\n\tat net.sf.saxon.expr.instruct.ApplyTemplates.processLeavingTail(ApplyTemplates.java:252)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)\r\n\tat net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)\r\n\tat net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)\r\n\tat net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)\r\n\tat net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)\r\n\tat net.sf.saxon.trans.TextOnlyCopyRuleSet.process(TextOnlyCopyRuleSet.java:66)\r\n\tat net.sf.saxon.trans.Mode.applyTemplates(Mode.java:478)\r\n\tat net.sf.saxon.Controller.transformDocument(Controller.java:2402)\r\n\tat net.sf.saxon.Controller.transform(Controller.java:1970)\r\n\tat net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:596)\r\n\tat net.sf.saxon.jaxp.TransformerImpl.transform(TransformerImpl.java:73)\r\n\tat org.takes.rs.RsXslt.transform(RsXslt.java:160)\r\n\tat org.takes.rs.RsXslt.transform(RsXslt.java:131)\r\n\t... 45 more\r\n</pre>\n    </body>\n</html>\n"
  TkProfileTest.rendersProfilePageWithoutRateInFirefox:101 
Expected: a string containing "rate</a> is not defined"
     but: was "<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <meta charset="utf-8"/>
        <title>0crat: error</title>
        <meta name="description" content="Internal application error"/>
        <meta name="viewport" content="width=device-width,initial-scale=1.0"/>
        <link rel="shortcut icon" href="/png/logo.png"/>
        <link rel="stylesheet" href="/css/main.css"/>
        <title>oops...</title>
    </head>
    <body>
        <p>
            <a href="/">
                <img src="/svg/logo.svg" class="logo" />
            </a>
        </p>
        <h1>Internal application error</h1>
        <p>1.0-SNAPSHOT, </p>
        <p>
            Please, report it
            <a href="https://github.com/zerocracy/farm/issues">here</a>.
            We will do our best to fix it as soon as possible.
            We're very sorry.
        </p>
        <pre>java.lang.IllegalStateException: [GET http://www.example.com/u/yegor256] failed in 289ms: ValidationException: Cannot convert string &quot;-1,000&quot; to double
	at org.takes.facets.fallback.TkFallback.error(TkFallback.java:234)
	at org.takes.facets.fallback.TkFallback.access$100(TkFallback.java:54)
	at org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:202)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsPrint.printBody(RsPrint.java:161)
	at org.takes.rs.RsPrint.printBody(RsPrint.java:91)
	at com.zerocracy.tk.profile.TkProfileTest.firefoxView(TkProfileTest.java:128)
	at com.zerocracy.tk.profile.TkProfileTest.rendersProfilePageWithoutRateInFirefox(TkProfileTest.java:102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: ValidationException: Cannot convert string &quot;-1,000&quot; to double
	at org.takes.rs.RsXslt.transform(RsXslt.java:133)
	at org.takes.rs.RsXslt.access$100(RsXslt.java:89)
	at org.takes.rs.RsXslt$1.body(RsXslt.java:113)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.fork.RsFork$1.body(RsFork.java:74)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at com.zerocracy.tk.TkSoftForward$RsSoft$1.body(TkSoftForward.java:95)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.forward.TkForward$Safe.load(TkForward.java:116)
	at org.takes.facets.forward.TkForward$Safe.body(TkForward.java:102)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.fork.RsFork$1.body(RsFork.java:74)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeader$1.body(RsWithHeader.java:114)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.rs.RsWithHeaders$1.body(RsWithHeaders.java:78)
	at org.takes.rs.RsWrap.body(RsWrap.java:65)
	at org.takes.facets.fallback.TkFallback$2.body(TkFallback.java:187)
	... 23 more
Caused by: ValidationException: Cannot convert string &quot;-1,000&quot; to double
	at net.sf.saxon.type.ValidationFailure.makeException(ValidationFailure.java:393)
	at net.sf.saxon.type.ValidationFailure.asAtomic(ValidationFailure.java:419)
	at net.sf.saxon.expr.sort.UntypedNumericComparer.quickComparison(UntypedNumericComparer.java:159)
	at net.sf.saxon.expr.sort.UntypedNumericComparer.quickCompare(UntypedNumericComparer.java:59)
	at net.sf.saxon.expr.GeneralComparison.compare(GeneralComparison.java:833)
	at net.sf.saxon.expr.GeneralComparison.evaluateManyToOne(GeneralComparison.java:711)
	at net.sf.saxon.expr.GeneralComparison.effectiveBooleanValue(GeneralComparison.java:660)
	at net.sf.saxon.expr.instruct.Choose.choose(Choose.java:902)
	at net.sf.saxon.expr.instruct.Choose.iterate(Choose.java:953)
	at net.sf.saxon.expr.instruct.BlockIterator.next(BlockIterator.java:49)
	at net.sf.saxon.tree.iter.AdjacentTextNodeMergingIterator.next(AdjacentTextNodeMergingIterator.java:52)
	at net.sf.saxon.tree.iter.UntypedAtomizingIterator.next(UntypedAtomizingIterator.java:50)
	at net.sf.saxon.tree.iter.UntypedAtomizingIterator.next(UntypedAtomizingIterator.java:32)
	at net.sf.saxon.expr.ItemMappingIterator.next(ItemMappingIterator.java:108)
	at net.sf.saxon.functions.FoldingFunction.call(FoldingFunction.java:59)
	at net.sf.saxon.expr.FunctionCall.iterate(FunctionCall.java:547)
	at net.sf.saxon.expr.Expression.evaluateItem(Expression.java:831)
	at net.sf.saxon.expr.Expression.evaluateAsString(Expression.java:890)
	at net.sf.saxon.expr.instruct.SimpleNodeConstructor.processLeavingTail(SimpleNodeConstructor.java:218)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.NamedTemplate.expand(NamedTemplate.java:262)
	at net.sf.saxon.expr.instruct.CallTemplate.process(CallTemplate.java:353)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)
	at net.sf.saxon.expr.instruct.ApplyTemplates.apply(ApplyTemplates.java:295)
	at net.sf.saxon.expr.instruct.ApplyTemplates.processLeavingTail(ApplyTemplates.java:252)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.Block.processLeavingTail(Block.java:687)
	at net.sf.saxon.expr.instruct.Instruction.process(Instruction.java:151)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:337)
	at net.sf.saxon.expr.instruct.ElementCreator.processLeavingTail(ElementCreator.java:284)
	at net.sf.saxon.expr.instruct.TemplateRule.applyLeavingTail(TemplateRule.java:347)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:502)
	at net.sf.saxon.trans.TextOnlyCopyRuleSet.process(TextOnlyCopyRuleSet.java:66)
	at net.sf.saxon.trans.Mode.applyTemplates(Mode.java:478)
	at net.sf.saxon.Controller.transformDocument(Controller.java:2402)
	at net.sf.saxon.Controller.transform(Controller.java:1970)
	at net.sf.saxon.s9api.XsltTransformer.transform(XsltTransformer.java:596)
	at net.sf.saxon.jaxp.TransformerImpl.transform(TransformerImpl.java:73)
	at org.takes.rs.RsXslt.transform(RsXslt.java:160)
	at org.takes.rs.RsXslt.transform(RsXslt.java:131)
	... 45 more
</pre>
    </body>
</html>
"
Tests in error: 
  TkProfileTest.rendersProfilePageWithRateInFirefox:81 » CashParsing Can't parse...
```
The full build log is here: https://gist.github.com/proshin-roman/74a020cc2beb2f7751628ddcace85cda

Not sure if it's locale-specific issue - maybe somebody else can try to build the project on Windows too.

--- a/pom.xml
+++ b/pom.xml
@@ -1018,6 +1018,9 @@ SOFTWARE.
           <plugin>
             <groupId>org.jacoco</groupId>
             <artifactId>jacoco-maven-plugin</artifactId>
+            <configuration>
+              <skip>false</skip>
+            </configuration>
             <executions>
               <execution>
                 <configuration>
@@ -1029,6 +1032,8 @@ SOFTWARE.
                     <exclude>com/zerocracy/pm/cost/Equity.java</exclude>
                     <exclude>com/zerocracy/tk/TkApp.java</exclude>
                     <exclude>com/zerocracy/tk/profile/TkIdentify.java</exclude>
+                    <exclude>com/zerocracy/tk/profile/TkProfile.java</exclude>
+                    <exclude>com/zerocracy/tk/TkBoard.java</exclude>
                   </excludes>
                 </configuration>
               </execution>

--- a/pom.xml
+++ b/pom.xml
@@ -1007,6 +1007,8 @@ SOFTWARE.
                   <exclude>com.zerocracy.pm.cost.EquityTest</exclude>
                   <exclude>com.zerocracy.tk.TkAppTest</exclude>
                   <exclude>com.zerocracy.tk.profile.TkIdentifyTest</exclude>
+                  <exclude>com.zerocracy.tk.profile.TkProfileTest</exclude>
+                  <exclude>com.zerocracy.tk.TkBoardTest</exclude>
                 </excludes>
               </configuration>
             </plugin>

